src/main/java/org/candlepin/controller/PoolManager.java

```diff
@@ -133,7 +133,7 @@ public PoolManager(
         EntitlementCurator entitlementCurator,
         ConsumerCurator consumerCurator,
         ConsumerTypeCurator consumerTypeCurator,
-        EntitlementCertificateGenerator ecGenerator,
+        EntitlementCertificateService ecGenerator,
```

Comment:
We can rename this from `ecGenerator` now that the class name has changed

This also applies to:
- ProductManager.java
- ConsumerResource.java
- EnvironmentResource.java
- RegenEnvEntitlementCertsJobTest.java
- RegenProductEntitlementCertsJobTest.java
- EntitlementCertificateServiceTest.java
- PoolManagerFunctionalTest.java
- PoolManagerTest.java
- ProductManagerTest.java
- AnonymousConsumerResourceCreationTest.java
- ConsumerContentOverrideResourceTest.java
- ConsumerResourceCreationTest.java
- ConsumerResourceTest.java
- ConsumerResourceUpdateTest.java
- EnvironmentResourceTest.java

Reply:
Done.